apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -287,6 +287,28 @@ export default class Utilities {
     });
   }
 
+  public static executeShellCommand(
```

Comment:
Agreed. Not sure why we need this command?

---
In reply to: [116886431](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r116886431) [](ancestors = 116886431)